lxc/image.go

```diff
@@ -288,14 +288,28 @@ func (c *imageCmd) run(config *lxd.Config, args []string) error {
 			inName = "default"
 		}
 
+		imageNames := []string{inName}
+
+		if len(args) > 2 {
+			for _, extraImage := range args[2:] {
+				imageNames = append(imageNames, extraImage)
+			}
+		}
```

Comment:
My understanding is that args[1] could contain a remote name, so it's passed through config.ParseRemoteAndContainer() to split the remote and image name. If we just iterate over the args[1:] slice, c.dereferenceAlias() will not like the remote name if found in first argument, and we'll get a "not found" error.

In this piece of code, instead of the for .. range loop, I could just do imageNames = append(imageNames, args[2:]...) though. But note, the slice is args[2:] not args[1:] since first arg is still handled specially a few lines above.

Let me know if I'm mis-understanding this, and I'll make your proposed changes.